### ISSUE TITLE
After leaving app and reentering, vibration no longer works.

### ISSUE THREAD

**Issue by nanogram** [iNDS-Team/iNDS#73]
After double tapping home button and entering another app, and then dub tapping the button again and reentering iNDS, the vibration feedback for pushing emulator controls no longer works. Only fixed by restarting iNDS.

**Comment by FrederickGeek8** [comment]
I was unable to reproduce on the slightly out-of-date version of iNDS on my primary device. I will test with a more recent build later today.

**Comment by scott0123** [comment]
@FrederickGeek8 Were you able to reproduce this?
I suspect that its caused by recording and AudioServicesPlaySystemSoundWithVibration not co-existing nicely: https://github.com/iNDS-Team/iNDS/pull/69#issuecomment-441108860
In which case we need to make sure the mic settings are properly reapplied upon a resumed suspension.

**Comment by FrederickGeek8** [comment]
I was able to reproduce on iNDS 1.10.3. I'm looking into it now.

**Comment by scott0123** [comment]
Yes I've logged the events after reproducing the bug and it seems to me that the AudioServicesPlaySystemSoundWithVibration line is still being run but produces no effect, among other things.

When we interrupt the app by going to menu we suspend a bunch of processes that are running and when we go back to the app we try to resume those same processes. This leads me to believe that some parts (probably audio related) need to be re-initialized when we un-suspend said processes.

As a last resort, if you know the bug didn't occur prior to some recent release, you can try to revert back and see what the problematic commit was and try to narrow down the bug that way.
